src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40 * days;
+
+  switch (true) {
+    case days >= 3 && days < 7: {
+      return rentalCost - 20;
+    }
+
+    case days >= 7: {
+      return rentalCost - 50;
+    }
+  }
```

Comment:
Prefer using 'if' with 'return' over 'switch' for boolean conditions to simplify the logic. Also, avoid adding 'else' after an 'if' with a 'return' as the code after it won't be executed anyway.